eslint-plugin-kuc-v1/no-sorting-in-alphabetical-order.js

```diff
@@ -60,6 +60,32 @@ module.exports = {
         )
         .join("\n");
     }
+    function checkNameOrder(typeAnnotation, node){
+      const props = typeAnnotation.members;
+      if(!props){
+        return;
+      }
+      const propInfos = [];
+      props.forEach(prop => {
```

Comment:
I think we can check data is not empty, before get value
this code `typeAnnotation.range[0]` maybe error when there is no range[0]
ex: 
this is my code, you can view and double check
```
props.forEach(prop => {
        if(!prop) return;
        if (prop.type !== "TSPropertySignature") return;
        if(!prop.typeAnnotation) return;
        if(!prop.typeAnnotation.typeAnnotation) return;

        const typeAnnotation = prop.typeAnnotation.typeAnnotation;
        let range = [];
        if(typeAnnotation.range && typeAnnotation.range[0]) range = typeAnnotation.range[0];
        if(range.length < 1) return;

        const typeValue = `${
          sourceCode.getTokenByRangeStart(range).value
        }_${typeAnnotation.type}`;

        propInfos.push({ name: prop.key.name, type: typeValue });
  });

let wrongOrderNames = [];
if(propInfos.length > 0) wrongOrderNames = checkNames(propInfos);
```